core/unit_test/CMakeLists.txt

```diff
@@ -759,6 +759,15 @@ if(Kokkos_ENABLE_CUDA)
     ${Cuda_SOURCES_SHAREDSPACE}
   )
 
+  # Suppress warning:
+  #   "Stack size for entry function 'xxx' cannot be statically determined"
+  # See https://github.com/kokkos/kokkos/issues/2039.
+  if(KOKKOS_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE)
+    target_link_options(Kokkos_CoreUnitTest_Cuda3
```

Comment:
@dalg24 Do we want this ?

Reply:
Seems not, see https://github.com/kokkos/kokkos/pull/7455#discussion_r1805362482. Closing this comment in favor of the mentioned one.